bin/ghe-restore

```diff
@@ -271,7 +277,7 @@ fi
 # Update the remote status to "complete". This has to happen before importing
 # ssh host keys because subsequent commands will fail due to the host key
 # changing otherwise.
-trap "" EXIT
+trap "" EXIT 1 2 3 15
```

Comment:
Sure does, my development system is using _dash_ which doesn't support using the SIG prefixes symbolic names.